sources/Browser/Core/iTermBrowserManager.swift

```diff
@@ -563,6 +705,37 @@ class iTermBrowserManager: NSObject, WKURLSchemeHandler, WKScriptMessageHandler
         }
     }
 
+    func setTransparency(_ value: Double) {
+        DLog("setTransparency called with value: \(value)")
+        Task { @MainActor in
+            let bgColor = value == 1.0 ? "transparent" : "rgba(0,0,0,\(1.0 - value))"
+            let script = """
```

Comment:
Move to its own .js file and use template substitutions rather than interpolated string substitution